packages/tailwindcss/src/compat/apply-config-to-theme.test.ts

```diff
@@ -1,9 +1,9 @@
-import { test } from 'vitest'
+import { expect, test } from 'vitest'
 import { buildDesignSystem } from '../design-system'
 import { Theme } from '../theme'
 import { applyConfigToTheme } from './apply-config-to-theme'
 
-test('Config values can be merged into the theme', ({ expect }) => {
```

Comment:
I noticed that we are inconsistent here with wether we capture `expect` from the `vi` module or from the test callback so I updated (hopefully) all places to match the more-commonly-used style

Reply:
I don't think we have any concurrent tests in this file but if any async tests in a given file run concurrently you have to use expect from the test function for error messages to be correct.

Reply:
Hmmm maybe this is no longer needed (thanks to async local storage or so)? I got curious and checked but both cases have the same output: 

<img width="1023" alt="Screenshot 2024-09-13 at 11 15 01" src="https://github.com/user-attachments/assets/f708bd31-2ab5-4558-bdd4-9e68a47e6fd0">
<img width="1117" alt="Screenshot 2024-09-13 at 11 15 18" src="https://github.com/user-attachments/assets/4a94ce0c-3f07-492f-b65c-764b01dd0ef5">

Reply:
Ah it looks like this only applies to `expect.assertions(…)` now: https://vitest.dev/api/expect.html#expect-assertions

So as long as that is not the thing being used then it's probably fine.